ftl/core/deck-config.ftl

```diff
@@ -104,22 +104,22 @@ deck-config-leech-action-tooltip =
 ## Burying section
 
 deck-config-bury-title = Burying
-deck-config-bury-siblings = Bury siblings
-deck-config-do-not-bury = Do not bury siblings
-deck-config-bury-if-new = Bury if new
-deck-config-bury-if-new-or-review = Bury if new or review
-deck-config-bury-if-new-review-or-interday = Bury if new, review, or interday learning
-deck-config-bury-tooltip =
-    Siblings are other cards from the same note (eg forward/reverse cards, or
-    other cloze deletions from the same text).
-    
-    When this option is off, multiple cards from the same note may be seen on the same
-    day. When enabled, Anki will automatically *bury* siblings, hiding them until the next
-    day. This option allows you to choose which kinds of cards may be buried when you answer
-    one of their siblings.
-    
-    When using the V3 scheduler, interday learning cards can also be buried. Interday
-    learning cards are cards with a current learning step of one or more days.
+deck-config-bury-new-siblings = Bury new siblings
+deck-config-bury-review-siblings = Bury review siblings
+deck-config-bury-interday-learning-siblings = Bury interday learning siblings
+deck-config-bury-new-tooltip =
+    Whether other `new` cards of the same note (eg reverse cards, adjacent cloze deletions)
+    will be delayed until the next day.
+deck-config-bury-review-tooltip = Whether other `review` cards of the same note will be delayed until the next day.
+deck-config-bury-interday-learning-tooltip =
+    Whether other `learning` cards of the same note with intervals > 1 day
+    will be delayed until the next day.
+deck-config-bury-priority-tooltip =
+    Which one of multiple siblings will be shown if burying is enabled, depends on the *gather*
+    order. The card gathered first will be shown, its siblings will be buried.
+    Since learning cards are gathered first, then review cards, and finally new cards,
+    a review card can never be buried by a new sibling. So if you enable burying new cards,
+    but not review cards, you may see a new and a review sibling in the same session.
```

Comment:
A bit lengthy perhaps. Please tweak it to your liking.

Reply:
Does the new wording work for you?

Reply:
I'm going to assume that's a yes. Thanks for your work on this Rumo :-)